docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,286 @@
+
+This section gives an overview of the transformations (ignores) that are available to you. Below is an overview table followed by explanation and examples for every transformation.
+
+.. list-table:: Overview
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Options
+     - Description
+   * - ignoreRegularExpressionKey
+     - Replaces all occurrences of regular expression with
+      
+       IGNORE
+   * - ignoreKey 
+     - Replaces all matching occurrences of provided key with
+      
+       IGNORE  
+   * - ignoreContentBetweenKeys 
+     - Replaces all characters found between key1 and key2,
+      
+       excluding key1 and key2, with IGNORE
+   * - ignoreKeysAndContentBetweenKeys
+     - Replaces all characters found between key1 and key2, 
+       
+       including key1 and key2, with IGNORE
+   * - ignoreContentBeforeKey
+     - Ignores everything before provided key has been found
+   * - ignoreContentAfterKey
+     - Ignores everything after provided key has been found
+   * - ignoreCurrentTimeBetweenKeys       
+     - Ignore current time between key1 and key2
+   * - replaceRegularExpressionKeys
+     - Replaces all matches of regular expression (key1) with
+      
+       the value of key2
+   * - replaceKey
+     - Replaces all matches occurrences of provided key with
+      
+       provided value
+   * - replaceEverywhereKey     
+     - Replaces all matches occurrences of provided key with
+      
+       provided value, same as replaceKey
+   * - removeRegularExpressionKey
+     - Replaces all matches of regular expression with “”
+   * - removeKeysAndContentBetweenKeys
+     - Removes all characters found between key1 and key2,
+      
+       including key1 and key2
+   * - removeKey 
+     - Removes all matching occurrences of provided key
+   * - decodeUnzipContentBetweenKeys
+     - Decode and unzip content between key1 and key2
+   * - canonicaliseFilePathContentBetweenKeys
+     - Canonicalise filepath content between key1 and key2
+   * - formatDecimalContentBetweenKeys
+     - Format decimal content between key1 and key2
+
+
+Usually those ignores are written in the ``common.properties`` file where you define your services (see section :ref:`testingLarvaServices`), in this way they apply to all test scenarios which have included this ``common.properties`` file. If you want to use ignores in a particular test scenario, then you can put them in that related ``scenario.properties`` file.
+
+Some ignores require 2 keys to indicate start and end, which are denoted as key1 and key2. Both require the same identifier. The identifier can be either an increasing sequence of integers starting at 1 for each ignore or an string starting with a “.”. The text indicates what to look for.
+
+Imagine that you want to compare the following result with the expected result:
+
```

Comment:
The way you introduce the example XML suggests that you are going to explain something about it right here. I was surprised to see the section header after the XML. Can you introduce it differently? Maybe something like: "In the subsections below, the ignores are explained by showing their effect on a target XML document. This target XML reads as follows:"

Reply:
I checked that this issue has been resolved. Thanks.